app/src/main/res/values/colors.xml

```diff
@@ -11,13 +11,13 @@
     <color name="grey_light">#D9D9D9</color>
     <color name="purple_default">#9F60FF</color>
     <color name="purple_light">#DAC2FF</color>
-    <color name="dark_01">#160039</color>
-    <color name="dark_02">#2E0072</color>
-    <color name="neutral_01">#191919</color>
-    <color name="neutral_02">#C7C7C7</color>
-    <color name="neutral_03">#DADADA</color>
-    <color name="neutral_04">#E4E4E4</color>
-    <color name="neutral_05">#F1F1F1</color>
+    <color name="dark_01_midnight_blue">#160039</color>
+    <color name="dark_02_blue">#2E0072</color>
+    <color name="neutral_01_great_dark_grey">#191919</color>
+    <color name="neutral_02_dark_grey">#C7C7C7</color>
+    <color name="neutral_03_grey">#DADADA</color>
+    <color name="neutral_04_light_grey">#E4E4E4</color>
+    <color name="neutral_05_lightest_grey">#F1F1F1</color>
     <color name="neutral_06">#FFFFFFFF</color>
     <color name="divider_primary">#F7CE7F</color>
```

Comment:
divider_primary რა ფერია შენც თუ გახსოვს

Reply:
ყვითელი, ჩატის გამყოფი ზოლის ფერი არ დამავიწყდება. ფიგმაზე დიზაინერებმა დაგვიწერეს ასე სახელი და მეორე ყვითელზე - ყვითელი. not my fault

Reply:
ა სორი მაშინ :დ